lnst/Common/IpAddress.py

```diff
@@ -126,6 +130,16 @@ def is_tentative(self):
         IFA_F_TENTATIVE = 0x40
         return IFA_F_TENTATIVE & self.flags
 
+
+def interface_addresses(network: Union[IPv4Network, IPv6Network]) -> Generator[BaseIpAddress]:
+    """
+    Generator of BaseIpAddress used that iterates through all host addresses
```

Comment:
Extra word `... used ...`? Please fix or reword.

Reply:
Fixed let me know what you think, I will squash the commit for it later

Reply:
Re did the interface_address and NetworkParam docstrings let me know what you think.